Huion_Kamvas_GT191/index.md

```diff
@@ -0,0 +1,18 @@
+---
+VID: "256c"
+PID: "006e"
+vendor: Huion
+product: Kamvas GT191
+image: Huion_Kamvas_GT191
```

Comment:
Do you think you can find a basic image of this tablet, without any measurements overlaid? Just a photo?